docs/blog/2024-08-01-nx-19-5-update.md

```diff
@@ -0,0 +1,278 @@
+---
+title: 'Nx 19.5 is here! Stackblitz, Bun, Incremental Builds for Vite, Gradle Test Atomizer'
+slug: 'nx-19-5-adds-stackblitz-new-features-and-more'
+authors: ['Zack DeRose']
+cover_image: '/blog/images/2024-08-01/nx-19-5-thumbnail.png'
+tags: [nx, release]
+---
+
+## Table of Contents
+
+In this blog post:
+
+- [Announcing Nx Cloud Hobby Tier](#announcing-nx-cloud-hobby-tier)
+- [StackBlitz Support](#stackblitz-support)
+- [Bun and Pnpm v9 Support](#bun-and-pnpm-v9-support)
+- [Local Flaky Task Detection](#local-flaky-task-detection)
+- [Project Detail View Enhancements](#project-detail-view-enhancements)
+- [Pattern Support for `targetDefaults`](#pattern-support-for-targetdefaults)
+- [Individual Targets Can Now Opt Out of Parallelism](#individual-targets-can-now-opt-out-of-parallelism)
+- [Support For Incremental Builds For Vite](#support-for-incremental-builds-for-vite)
+- [Project Crystal Conversion Generators](#project-crystal-conversion-generators)
+- [Gradle Composite Builds Support](#gradle-composite-builds-support)
+- [Experimental: Gradle Test Atomization](#experimental-gradle-test-atomization)
+- [Experimental: Nx Release Adds Version Plans Support](#experimental-nx-release-adds-version-plans-support)
+- [Support for React 19 (rc) and Angular 18.1](#support-for-react-19-rc-and-angular-181)
+- [Automatically Update Nx](#automatically-update-nx)
+- [Monorepo World Conference Speakers Announced!!](#monorepo-world-conference-speakers-announced)
+
+## Announcing Nx Cloud Hobby Tier
+
+We just added a new [Hobby Tier](/pricing#plan-details) to Nx Cloud! Why is this exciting? It is completely free and contains all major features, which includes the ability to [distribute your tasks with Nx Agents](/ci/features/distribute-task-execution).
+
+This gives you plenty of opportunities to experiment with Nx Cloud on your workspace and see its impact on your teams productivity.
+
+[![Nx Cloud Tiers](/blog/images/2024-08-01/start-with-everything-scale-when-you-need.jpg)](/pricing#plan-details)
+
+Checkout the [plan details page](/pricing#plan-details) for more info, and see how Nx Cloud can help you! And in case you missed it, check out our latest blog on the latest Nx Cloud feature: [Explain with AI](/blog/explain-with-ai)!
+
+## StackBlitz Support
+
+Nx now has support for [Stackblitz](https://stackblitz.com/) (again). For a while it was not possible to use Nx on Stackblitz because we implemented some perf critical parts of Nx as native components with Rust. Huge shoutout to the Stackblitz team and in particular [LongYinan](https://github.com/Brooooooklyn) for [their amazing work of pushing napi-rs](https://napi.rs/) forward and helping us [add WebAssembly support to Nx](https://github.com/nrwl/nx/pull/22870).
```

Comment:
```suggestion
Nx now has support for [Stackblitz](https://stackblitz.com/) (again). For a while it was not possible to use Nx on Stackblitz because we implemented some perf critical parts of Nx as native components with Rust. Huge shoutout to [LongYinan](https://github.com/Brooooooklyn) for [their amazing work of pushing napi-rs](https://napi.rs/) forward and helping us [add WebAssembly support to Nx](https://github.com/nrwl/nx/pull/22870) as well as the StackBlitz team who helped debug issues.
```